### DISCUSSION TITLE
Last.fm scrobbler incorrectly states last.fm login details are incorrect

### DISCUSSION THREAD

**Discussion by beneix** [LMS-Community/slimserver#844] [General]
I used last.fm scrobbling years ago and I thought it was still active. Discovered it wasn't and went to re-enter my last.fm details. LMS claimed that they were incorrect, even though I had just confirmed on last.fm website that they worked. This leads me to believe that there are limitations in terms of which characters can be entered in the password field in the LMS settings interface, since I have a special character in my last.fm password.

The error message I get is:
`There was an error verifying your account with Last.fm. Please try again later. (Invalid Last.fm username or password.)`

Until there is a fix, is there a way to enter the password in a settings file somewhere by logging into my LMS via SSH? I am running a PiCorePlayer server.

**Comment by michaelherger** [comment]
What would that character be?

**Reply by beneix** [reply]
I have an '@' in the password.

**Reply by michaelherger** [reply]
Like in `p@ssw0rd`? 😂 

Could you please test without that character? Only to prove the suspicion? And are you using your username or your email as the username?

**Reply by beneix** [reply]
Doh! I was trying with email instead of username. It's been too long a hiatus since I used Last.fm...apologies for the false alarm. With my username, it all works. This issue is resolved.
